pkg/views/template.go

```diff
@@ -18,13 +18,13 @@ type Template struct {
 func NewTemptale() Template {
 	var t Template
 	template, err := template.ParseFiles(
```

Comment:
to avoid that renaming you can pass template folder location as arguments in this function.
Path to the file could be create like this: ```path.Join(folder, "index.html")```